packages/appeals-service-api/src/mappers/questionnaire-submission/has-mapper.js

```diff
@@ -0,0 +1,87 @@
+class HasQuestionnaireMapper {
+	mapToPINSDataModel(journeyResponse) {
+		return [
+			{
+				questionnaire: {
+					LPACode: journeyResponse.LPACode,
+					caseReference: journeyResponse.caseReference,
+					isAppealTypeAppropriate: journeyResponse.answers['correct-appeal-type'] ? true : false,
+					doesTheDevelopmentAffectTheSettingOfAListedBuilding: journeyResponse.answers[
+						'affects-listed-building'
+					]
+						? true
+						: false,
+					affectedListedBuildings: () => {
+						if (journeyResponse.answers['affects-listed-building']) {
+							return journeyResponse.answers['listed-building-number'];
```

Comment:
Do we need a bit of logic to transform this data to conform to the PINS data model (e.g without the id we attach to it?)

Reply:
It could maybe be a utility function to remove the id from any list add more form response?